docs/community/100-days-of-oss.md

```diff
@@ -22,8 +22,15 @@ There are numerous ways to participate in the #100DaysOfOSS challenge, including
 - create or participate in discussions related to OSS topics;
 - write a blog post or create content that supports an OSS project; 
```

Comment:
[nitpick]Since you've listed these two points, I recommend omitting this part for organizational reasons. 
```suggestion 
```

Reply:
Okay